backend/plugins/mongolian_api/src/modules/ebarimt/graphql/subscriptionPlugin.js

```
@@ -0,0 +1,24 @@
+var { withFilter } = require('graphql-subscriptions');
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)